### PR TITLE
test: use categories in post

### PR DESCRIPTION
**Pull request by jekyllbot**
PR automatically created for @DirtyF.

```text
test: use categories in post
```

### CODE REVIEW

File: spec/fixtures/_posts/2014-03-04-march-the-fourth.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ tags:
  - '"/><VADER>'
 image:
   path: "/object-image.png"
-category: updates
+categories: updates jekyll
```
Comment (pathawks) [inline spec/fixtures/_posts/2014-03-04-march-the-fourth.md]: Does this need to be an `[Array]`?

Comment (DirtyF) [reply spec/fixtures/_posts/2014-03-04-march-the-fourth.md]: Both notations works, default post uses this syntax:

https://github.com/jekyll/jekyll/blob/master/lib/site_template/_posts/0000-00-00-welcome-to-jekyll.markdown.erb#L5

File: spec/jekyll-feed_spec.rb [hunk 2]
```diff
@@ -43,7 +43,8 @@
   end
 
   it "puts all the posts in the feed.xml file" do
-    expect(contents).to match "http://example.org/updates/2014/03/04/march-the-fourth.html"
+
```
Comment (ashmaroli) [inline spec/jekyll-feed_spec.rb]: Unnecessary empty line..
```suggestion

```

**Comment by ashmaroli** [comment]
@jekyllbot: merge +dev

**Pull request merged**